source/includes/_resource.customers.md

```diff
@@ -132,6 +137,17 @@ vhx.customers.create({
       </td>
       <td>Billing parameters for <a href="/in-app-purchases">In-App Purchases</a> for the Apple, Google, and Roku platforms.</td>
     </tr>
+    <tr class="text-2 border-bottom border--light-gray">
+      <td class="nowrap">
+        <strong class="is-block text--navy">plan</strong>
+        <span class="is-block text--transparent text-3">string</span>
+        <span class="text--transparent text-3">optional</span>
+      </td>
+      <td>
+      The tier of content the customer will be able to access. Values can be one of
```

Comment:
Let's maybe stick with "plan" instead of "tier". That said, not sure the best way to re-word this.